javascript/code-challenges/challenges-10.test.js

```diff
@@ -71,9 +91,24 @@ const alkiBeach = [33, 31, 147, 130, 27, 93, 38, 126, 141, 63, 46, 17];
 const cookieStores = [firstPike, seaTac, seattleCenter, capHill, alkiBeach];
 
 const grandTotal = (stores) => {
-  // Solution code here...
+  
+  let hourlySalesArr = [];
+  let hourlySales = 0;
+
+  for( let i=0; i < hoursOpen.length; i++) {
+    for (let j = 0; j<stores.length; j++) {
+      hourlySales += stores[j][i];
+    }
+  
+    hourlySalesArr.push(hourlySales);
+
+    hourlySales = 0;
+  }
+
+    return hourlySalesArr;
 
-};
+
+  };
```

Comment:
Really nice work on the challenges!

When you have time at some point, I would see if you can incorporate a higher order function for this one (perhaps map() and forEach()) 👍